galaxy_ng/app/api/ui/serializers/collection.py

```diff
@@ -155,6 +145,7 @@ def get_all_versions(self, obj):
         versions_in_repo = self._get_versions_in_repo(obj)
         return CollectionVersionSummarySerializer(versions_in_repo, many=True).data
 
+    # TODO(awcrosby): rename field to "version_details" since with
+    # "version" query param this won't always be the latest version
```

Comment:
Or possibly in the future, just always return a list for 'version' and let clients assume the first item is the highest or best version.